cmd/ceremony/cert.go

```diff
@@ -271,6 +287,8 @@ func makeTemplate(randReader io.Reader, profile *certProfile, pubKey []byte, ct
 		cert.IsCA = false
 	case crlCert:
 		cert.IsCA = false
+	case requestCert:
+		fallthrough
 	case intermediateCert:
```

Comment:
nit: could be `case intermediateCert, requestCert:`, since fallthrough is really intended to do whatever this case says _and_ whatever the next case says, but in this instance the `requestCert` case is empty.

Reply:
I'm going to go ahead and merge, and send this as a followup.